cb_defense_syslog.py

```diff
@@ -360,49 +346,49 @@ def parse_notifications(server, notifications_response, audit_response):
     if notifications_response != None and audit_response != None:
 
         if config.get('general', 'output_format').lower() == 'json':
-            audit_log = al.parse_response_json(audit_response, source, logger, get_unicode_string)
-            if threat_hunter:
-                notifications_log=n.parse_response_json_threathunter(notifications_response, source, logger, get_unicode_string)
+            audit_log = al.parse_response_json(audit_response, source, get_unicode_string)
+            if threat_hunter is True:
```

Comment:
Boolean evaluates as conditional expressions so you don't need a comparison to the boolean value.

Reply:
Thanks Becca! I wasn't sure how the Conditional reacts to a Boolean vs None which was why I had the **is True**. I have made those changes and will be incorporated in the next push.